core/include/detray/builders/grid_factory.hpp

```diff
@@ -308,10 +380,21 @@ class grid_factory {
         constexpr auto e_phi_axis = static_cast<dindex>(axes_t::label1);
 
         auto b_values = grid_bounds.values();
+        // Overwrite the mask values if axis spans are provided
+        if (axis_spans[0].size() > 1) {
+            b_values[boundary::e_inner_r] = axis_spans[0][0];
```

Comment:
Same here, is reduced phi coverage a good thing when the corresponding surface does not have phi boundaries?

Reply:
Removed.